app/javascript/dashboard/routes/dashboard/helpcenter/pages/categories/NameEmojiInput.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div
+    class="flex items-center relative"
+    @mouseover="showRemoveEmojiButton = true"
+    @mouseleave="showRemoveEmojiButton = false"
+  >
+    <div
+      v-if="icon && showRemoveEmojiButton"
+      class="absolute cursor-pointer z-20 top-4 ltr:left-11 rtl:right-11 rounded-xl"
+      @click="onClickInsertEmoji('')"
+    >
+      <fluent-icon
+        size="18"
+        icon="dismiss-circle"
+        type="outline"
+        class="text-red-400 dark:text-red-500 hover:text-red-700 dark:hover:text-red-300"
+      />
+    </div>
+    <div
+      class="flex items-center justify-center absolute w-14 z-10 hover:bg-slate-50 dark:hover:bg-slate-700 cursor-pointer h-[2.4375rem] rounded-[4px] top-[25px] border border-solid border-slate-200 dark:border-slate-800"
+      @click="toggleEmojiPicker"
+    >
+      <span v-if="icon" v-dompurify-html="icon" class="text-lg" />
+      <fluent-icon
+        v-else
+        size="18"
+        icon="emoji"
+        type="outline"
+        class="text-slate-800 dark:text-slate-100"
+      />
+    </div>
+    <woot-input
+      v-model.trim="name"
```

Comment:
Let's not use `.trim` here, instead let's trim it at the API. Having it here causes some weird issues when editing the text